phoenicis-scripts/src/main/java/org/phoenicis/scripts/nashorn/builtins/SetupWizardInjector.java

```diff
@@ -21,6 +21,8 @@ public SetupWizardInjector(UiSetupWizardFactory uiSetupWizardFactory) {
 
     @Override
     public void injectInto(NashornEngine nashornEngine) {
+        nashornEngine.eval("var InstallationType = Java.type(\"org.phoenicis.scripts.ui.InstallationType\")",
```

Comment:
I think it would be better to get the package at runtime, in case we move it later on.
Can we get this information through the `Class` object?